sicp-pict/main.rkt

```diff
@@ -290,18 +315,14 @@
      #'(begin (displayln 'painter-expr)
               (paint painter-expr))]))
 
-;;;
-;;; Syntactic Sugar
-;;;
-
 ; SYNTAX  (with-transformation transformation body ...)
-;   Store the initial-matrix of thed rawing context given by current-dc.
-;   Install  transformation  as the initial-matrix
-;   Evaluate body
+;   Store the initial-matrix of the drawing context given by current-dc.
+;   Install transformation as the initial-matrix
+;   Evaluate body ...
 ;   Restore the saved initial-matrix
 (define-syntax (with-transformation stx)
   (syntax-parse stx
-    [(_with-transformation transformation body ...)
+    [(_ transformation body ...)
      (syntax/loc stx
```

Comment:
This is purely stylistic choice. 

I prefer `(_with-transformation transformation body ...)` over `(_ transformation body ...)` 
since the pattern shows an actual use of the macro looks like.

But you don't need to change them back.